notebooks/recommender-system.ipynb

```diff
@@ -0,0 +1,843 @@
+{
```

Comment:
<p>How well did your prediction work? </p><p> </p><p>It might help to have a testing-training discussion. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cooperrc/uconn_numpy_tutorials/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>